src/components/CurriculumComponents/CurriculumVisualiserFilters/CurriculumVisualiserFiltersDesktop.tsx

```diff
@@ -0,0 +1,155 @@
+          </Box>
+          {threadOptions.map((threadOption) => {
+            const isSelected = isSelectedThread(threadOption);
+            const highlightedCount = highlightedUnitCount(
+              yearData,
+              selectedYear,
+              yearSelection,
+              selectedThread,
+            );
+
+            return (
+              <Box
+                $ba={1}
+                $background={isSelected ? "black" : "white"}
+                $borderColor={isSelected ? "black" : "grey40"}
+                $borderRadius={4}
+                $color={isSelected ? "white" : "black"}
+                $font={isSelected ? "heading-light-7" : "body-2"}
+                $ph={12}
+                $pt={12}
+                $mb={8}
+                key={threadOption.slug}
+              >
+                <RadioButton
+                  aria-label={threadOption.title}
+                  value={threadOption.slug}
+                  data-testid={
+                    isSelected ? "selected-thread-radio" : "thread-radio"
+                  }
+                >
+                  <OakSpan>
+                    {threadOption.title}
+                    <OakSpan aria-live="polite" aria-atomic="true">
+                      {isSelected && (
+                        <>
+                          <br />
+                          {highlightedCount}
+                          {highlightedCount === 1 ? " unit " : " units "}
+                          highlighted
+                        </>
+                      )}
+                    </OakSpan>
+                  </OakSpan>
+                </RadioButton>
+              </Box>
+            );
+          })}
+        </RadioGroup>
+      </Fieldset>
+      <Fieldset
+        $mr={16}
+        $mb={32}
+        $display={["none", "block"]}
+        data-testid="year-group-filter-desktop"
+      >
+        <FieldsetLegend $font={"heading-7"} $mb="space-between-xs">
+          Year group
+        </FieldsetLegend>
+        <RadioGroup
+          name="year"
+          value={selectedYear}
+          onChange={(e) => onSelectYear(e.target.value)}
+        >
+          <Box $mb={16}>
+            <RadioButton
+              aria-label="All year groups"
+              value={""}
+              data-testid={"all-years-radio"}
+            >
+              All
+            </RadioButton>
+          </Box>
+          {yearOptions.map((yearOption) => (
+            <Box key={yearOption} $mb={16}>
+              <RadioButton
+                value={yearOption}
+                data-testid={"year-radio"}
+                aria-label={getYearGroupTitle(yearData, yearOption)}
+              >
+                {getYearGroupTitle(yearData, yearOption)}
+              </RadioButton>
+            </Box>
+          ))}
+        </RadioGroup>
+        <ScreenReaderOnly aria-live="polite" aria-atomic="true">
+          Showing {unitCount} {unitCount === 1 ? "unit" : "units"}
+        </ScreenReaderOnly>
```

Comment:
```suggestion
```

As noted above, this also needs deleting in favour of the combined visually hidden live region that can be found at [UnitsTab.tsx L163-174](https://github.com/oaknational/Oak-Web-Application/pull/2908/files#diff-39b92cfb6d143330eb23c86f610efb127dbef7c59baed79a0ff1d526d2215231R163-R174) in this PR.